imap-proto/src/parser/rfc3501/mod.rs

```diff
@@ -534,7 +534,6 @@ pub fn parse_response(msg: &[u8]) -> ParseResult {
     response(msg)
 }
 
-
```

Comment:
This doesn't belong in this commit. Maybe some earlier commit made changes here? Otherwise a separate commit for this is also fine.